log/syslog_formatter.go

```diff
@@ -0,0 +1,84 @@
+package log
+
+import (
+	"fmt"
+	"log/syslog"
+	"os"
+
+	"github.com/Sirupsen/logrus"
+)
+
+var ceeTag = []byte("@cee:")
+
+var _ logrus.Formatter = &syslogger{}
+
+type syslogger struct {
+	wrap logrus.Formatter
+	out  *syslog.Writer
+}
+
+func newSyslogger(appname string, level string, fmter logrus.Formatter) (*syslogger, error) {
+	priority := getLocal(level)
+	out, err := syslog.New(priority, appname)
+	return &syslogger{
+		out:  out,
+		wrap: fmter,
+	}, err
+}
+
+func getLocal(level string) syslog.Priority {
+	switch level {
+	case "0":
+		return syslog.LOG_LOCAL0
+	case "1":
+		return syslog.LOG_LOCAL1
+	case "2":
+		return syslog.LOG_LOCAL2
+	case "3":
+		return syslog.LOG_LOCAL3
+	case "4":
+		return syslog.LOG_LOCAL4
+	case "5":
+		return syslog.LOG_LOCAL5
+	case "6":
+		return syslog.LOG_LOCAL6
+	case "7":
+		return syslog.LOG_LOCAL7
+	}
```

Comment:
This seems to be easier solved by a map than by running through a switch statement every time?

```go
var localMapping = map[string]syslog.Priority{
  "0": syslog.LOG_LOCAL0,
  "1": syslog.LOG_LOCAL1,
  // ...
}

func getLocal(level string) syslog.Priority {
  p, ok := localMapping[level]
  if !ok {
    return syslog.LOG_LOCAL0
  }
  return p
}
```